### ISSUE TITLE
Running on Raspberry Pi

### ISSUE THREAD

**Issue by driese21** [godarklight/DarkMultiPlayer#356]
Hey guys,

Not sure if the server on Linux is officially supported, but I've encountered a bug (or oddity) with the server time.
I'm not sure whether it affects something critical, but I'm just putting it here.
Here's the time on a Windows PC:
![correctsystemtime](https://cloud.githubusercontent.com/assets/6837614/11997026/627e6ac0-aa71-11e5-8a99-e2d7c7e832cc.PNG)
And here's the incorrect on my Pi:
![incorrrectsystemtime](https://cloud.githubusercontent.com/assets/6837614/11997028/6ceb39ca-aa71-11e5-84dc-fe376518e427.PNG)
![incorrrectsystemtime2](https://cloud.githubusercontent.com/assets/6837614/11997030/6fb19550-aa71-11e5-8c3c-25632b24af9d.PNG)

My Raspberry Pi version: Linux raspberrypi 4.1.13-v7+ with Raspbian GNU/Linux 8 (jessie)
Mono version info: Mono JIT compiler version 4.2.1 (Stable 4.2.1.102/6dd2d0d Thu Nov 12 10:06:49 UTC 2015)

If you need any more information, please let me know!

P.S.: apparently it has something to do with soft floating-point vs hard floating point (see this thread: http://stackoverflow.com/questions/16926634/datetime-now-tostring-weird-results)

**Comment by Stokkie64** [comment]
As far as I know DMP does not have explicit ARM support. It might support Linux (@RockyTV might know better) But the RasPi runs a ARM based CPU, meaning different architectures. So your FP vs HFP note makes sense on initial inspection. Some FP cast issue maybe.

Definitely something I have an interest in checking up on as I own a RasPi and love running these kinds of services on it.

I plan on spending more time on DMP in the new year once my studies start up again so will look into these things when the time comes

**Comment by godarklight** [comment]
Hrmm, DMP is sensitive to either of the system clocks changing after you connect, but after that it shouldn't be a problem.

I know the rPi doesn't have a real time clock, perhaps you managed to sync before the clock updated from NTP and then updated afterwards? (I'll have to test on my rPi2 later)

**Comment by driese21** [comment]
Well I first noticed the issue, and then checked my system time, it was one hour of (wrong timezone), then just adjusted it, let it sync to the right time, it was. After that I restarted the server, yet "problem" still persists. I'm not sure if this is game-breaking, but I'm hosting it on my Windows machine again.
I also tried pushing in a correct subspace.txt file (from the Windows machine) and that didn't help either.
